.eslintrc.js

```diff
@@ -1,280 +1,214 @@
 module.exports = {
-    'env': {
-        'browser': true,
-        'commonjs': true,
-        'es6': true
+    env: {
+        browser: true,
```

Comment:
there's eslint rule about this case - "quote-props"
currently it switched off, my offer to set it to: `"quote-props": [2, "as-needed", { "keywords": true }]`